packages/language-server/src/completion/completions.ts

```diff
@@ -877,6 +881,7 @@ function getSuggestionsForRelationDirective(
     .slice(0, position.character)
     .trim()
 
+  // If we are right after @relation(
```

Comment:
"right after" or "anywhere after"?

Reply:
It's `wordBeforePosition` so right after, but if you look closer there is also in the code `wordsBeforePosition` plural form and then it's the whole line.

Reply:
Hm but the `includes` could also mean that `@relation` is at the beginning of the string, and then some additional stuff comes between the match and our position, right?

Reply:
wordBeforePosition singular means it's only one word
Yes it can only happen for example with:
`@relation(onUpdate:...)` word = `@relation(onUpdate:` here (not sure if with or without ,)

Reply:
Ok, got it. Def not obvious.